src/x509_parser.rs

```diff
@@ -51,14 +51,23 @@ fn parse_rdn(i: &[u8]) -> BerResult<RelativeDistinguishedName> {
     .map(|(rem, x)| (rem, x.1))
 }
 
-pub(crate) fn parse_name(i: &[u8]) -> BerResult<X509Name> {
+pub fn parse_name(i: &[u8]) -> BerResult<X509Name> {
```

Comment:
I agree with making this public, but FYI I think I'll just change to a better name (`parse_x509_name`) by adding a commit

Reply:
Good idea!